### ISSUE TITLE
arsd.jni Possible to use D as jni in android studio?

### ISSUE THREAD

**Issue by Aphexus** [adamdruppe/arsd#454]
Do you think it would be possible to use D to write native code in android studio using arsd.jni to interface with kotlin?

**Comment by adamdruppe** [comment]
Yes, that should be doable. Actually, Android is why I wrote the jni lib in the first place (though I never actually used it for anything beyond a hello world there...) But you can compile to a .so and load it, i did helper programs for it https://code.dlang.org/packages/d_android with generated bindings to the android apis (but be warned, using them leads to BRUTALLY SLOW compile times) and it is years old so might not work anymore but still might help you get something going.
